vmngclient/vmanage_auth.py

```diff
@@ -0,0 +1,121 @@
+from typing import Optional
+from urllib.parse import urljoin
+
+import requests
+from requests import PreparedRequest
+from requests.auth import AuthBase
+from requests.cookies import RequestsCookieJar
+
+
+class InvalidCredentialsError(Exception):
+    """Exception raised for invalid credentials
+
+    Attributes:
+        username (str): vManage username.
+        password (str): vManage password.
+        message (str): precise error explanation.
+    """
+
+    def __init__(
+        self, username: str, password: str, message: str = "Username and/or password is incorrect. Please try again!"
```

Comment:
The message from vmanage is:
![image](https://user-images.githubusercontent.com/85410520/203798330-e4f434cb-5142-49c5-9e45-7e267cfff77f.png)
Shouldn't we include same options of invalid login?

Reply:
Yes, it wold be nice to have a negative test available.

Reply:
Moved to #45 Thanks!